src/slider/style.scss

```diff
@@ -37,25 +37,25 @@ tp-slider-nav {
 // Fade in Styles.
 tp-slider[behaviour="fade"] {
 
-	tp-slider-slides {
+	> tp-slider-track > tp-slider-slides {
```

Comment:
@imranhsayed Not sure about `> tp-slider-track` - any other solution? Do we need to make this update at all?

Reply:
@junaidbhura 
If we don't use the direct child then it adds those styles to nested slider slides also, while it should be only adding it to the parent slider.

Reply:
@imranhsayed Shouldn't this be applied to everything anwyway?

Reply:
No @junaidbhura It breaks the inner slider styles without this.

Reply:
@junaidbhura As discussed, I have made the updates.